app/src/main/java/com/breadwallet/tools/adapter/TransactionListAdapter.java

```diff
@@ -58,6 +60,13 @@ public TransactionListAdapter(Context mContext, List<TxItem> items) {
         this.syncingResId = R.layout.syncing_item;
         this.promptResId = R.layout.prompt_item;
         this.mContext = mContext;
+
+        /**
+         * current solution, just cached the result to [cachedOpsAll]
+         * previously, called multiple times inside `onBindViewHolder` and caused OOM
+         */
```

Comment:
Thanks for the note @andhikayuana .  Lets take this explanation out (remove lines 64 - 67) because this is open source code and it will lose context.  I have copied it into the PR